google/ads/google_ads/interceptors/interceptor.py

```diff
@@ -200,20 +196,19 @@ def _get_error_from_response(self, response):
                 # library-specific Error type for easy handling. These errors
                 # originate from the Google Ads API and are often caused by
                 # invalid requests.
-                self._exception = GoogleAdsException(
+                return GoogleAdsException(
                     response_exception, response, google_ads_failure,
                     request_id)
             else:
                 # Raise the original exception if not a GoogleAdsFailure. This
                 # type of error is generally caused by problems at the request
                 # level, such as when an invalid endpoint is given.
-                self._exception = response_exception
+                return response_exception
         else:
             # Raise the original exception if error has status code
             # INTERNAL or RESOURCE_EXHAUSTED, meaning that
```

Comment:
Are we missing text in the doc string? "meaning that" seems to be missing a clause.

Reply:
Fixed.